lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -16,27 +16,35 @@
   background-color: $nx-site-background-color;
   height: 100%;
   margin: 0;
-  // prevents scrollbar flicker as tooltips appear in IE11
-  overflow-y: hidden;
   padding: 0;
 }
 
 .nx-page {
-  @include container-vertical;
-
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  height: 100%;
+  display: grid;
   min-width: var(--page-width-min);
+  height: 100%;
+  width: 100%;
+
+  // NOTE: Most of this template is only used when using nx-page-content--full-width (or no nx-page-content at all).
+  // In traditional layouts that use plain nx-page-content, that element manages everything below the second
+  // row of this template.
+  grid-template:
+    'old-header     old-header  old-header' auto
+    'global-sidebar   notices      notices' auto
```

Comment:
So I see that we're defining the position of "notices" here as above the header but I cannot find any mockups that reflect this. @dturner32

Reply:
> So I see that we're defining the position of "notices" here as above the header but I cannot find any mockups that reflect this. @dturner32

I actually asked this morning in slack if my assumptions here are correct.  That conversation seems to have stalled out however